### PR TITLE
update the visibility output file

### PR DESCRIPTION
**Pull request by jaymedina**
- [x] make the downloadable visibility data file a `.csv`
- [x] make it so that the filename depends on the input parameters (`[targ]_[inst]_visibility.txt`)
- [x] add `.png` files for the `README.rst`
- [x] modified some text on the front end
- [x] fix NIRSpec bokeh plot issue

information to add in the `.csv` file:
- [x] ~a brief description of what the file is ~and what the columns represent~
- [x] target name 
- [x] instrument name
- [x] date that this file was created 
- [x] version of ExoCTK that made the file 
- [x] add a column with MJDs
- [x] add "," delimiter

### CODE REVIEW

**Comment by hover2pi** [comment]
@jaymedina is this ready for review? I see all check marks but still a [WIP]. Let me know!

**Comment by jaymedina** [comment]
ready for review

File: exoctk/exoctk_app/app_exoctk.py [hunk 1]
```diff
@@ -471,9 +471,32 @@ def contam_visibility():
             except:
                 pass
 
-            # Calculate
-            title = form.targname.data or ', '.join([form.ra.data, form.dec.data])
-            pG, pB, dates, vis_plot, table = vpa.using_gtvt(str(form.ra.data), str(form.dec.data), form.inst.data)
+
+                contamVars['visPA'] = True
```
Comment (hover2pi) [inline exoctk/exoctk_app/app_exoctk.py]: Nix this

File: exoctk/exoctk_app/app_exoctk.py [hunk 2]
```diff
@@ -471,9 +471,32 @@ def contam_visibility():
...
+                # Make plot
+                title = form.targname.data or ', '.join([form.ra.data, form.dec.data])
+                pG, pB, dates, vis_plot, table = vpa.using_gtvt(str(form.ra.data), str(form.dec.data), form.inst.data)
+          
+                # Make output table
+                vers = '0.3'
+                today = datetime.datetime.now()
+                fh = StringIO()
+                fh.write('# Hi! This is your Visibility output file for... \n')
+                fh.write('# Target: {} \n'.format(contamVars['tname']))
```
Comment (hover2pi) [inline exoctk/exoctk_app/app_exoctk.py]: contamVars['tname'] => form.targname.data

File: exoctk/exoctk_app/app_exoctk.py [hunk 3]
```diff
@@ -471,9 +471,32 @@ def contam_visibility():
...
+                fh.write('# Instrument: {} \n'.format(contamVars['inst']))
```
Comment (hover2pi) [inline exoctk/exoctk_app/app_exoctk.py]: contamVars['inst'] => form.inst.data

File: exoctk/exoctk_app/app_exoctk.py [hunk 4]
```diff
@@ -471,9 +471,32 @@ def contam_visibility():
             except:
                 pass
 
-            # Calculate
-            title = form.targname.data or ', '.join([form.ra.data, form.dec.data])
-            pG, pB, dates, vis_plot, table = vpa.using_gtvt(str(form.ra.data), str(form.dec.data), form.inst.data)
+
+                contamVars['visPA'] = True
+
+                # Make plot
```
Comment (hover2pi) [inline exoctk/exoctk_app/app_exoctk.py]: This block of code should be unindented once so it is inline with the try/except, not under the except loop.

File: exoctk/exoctk_app/app_exoctk.py [hunk 5]
```diff
@@ -510,10 +533,17 @@ def contam_visibility():
...
-                                   contam_js=contam_js, contam_css=contam_css)
+                return render_template('contam_visibility_results.html',
+                                       form=form, vis_plot=vis_div,
+                                       vis_table=visib_table,
+                                       vis_script=vis_script, vis_js=vis_js,
+                                       vis_css=vis_css, contam_plot=contam_div,
+                                       contam_script=contam_script,
+                                       contam_js=contam_js,
+                                       contam_css=contam_css,
+                                       tname=contamVars['tname'],
```
Comment (hover2pi) [inline exoctk/exoctk_app/app_exoctk.py]: Do you need to pass 'tname' and 'iname' to the template? You should be able to pull them from the form on the HTML page with `{{ form.targname.data }}` and `{{ form.inst.data }}`.

File: exoctk/exoctk_app/templates/contam_visibility_results.html [hunk 6]
```diff
@@ -83,7 +86,10 @@ <h3>Target Contamination</h3>
     	</form>
       <form action="/visib_result" method="post">
           <input type="hidden" name="data_file" value="{{ vis_table }}" />
-          <input class="btn btn-success" type='submit' value='Download TXT File'></p>
+          <input class="btn btn-success" type='submit' value='Download CSV File'></p>
+      <form action="/visib_result" method="post">
+          <input type="hidden" name="targetname" value="{{ tname }}" />
```
Comment (hover2pi) [inline exoctk/exoctk_app/templates/contam_visibility_results.html]: `{{ tname }}` => `{{ form.targname.data }}`

File: exoctk/exoctk_app/templates/contam_visibility_results.html [hunk 7]
```diff
@@ -83,7 +86,10 @@ <h3>Target Contamination</h3>
...
+          <input type="hidden" name="instrumentname" value="{{ iname }}" />
```
Comment (hover2pi) [inline exoctk/exoctk_app/templates/contam_visibility_results.html]: `{{ iname }}` => `{{ form.inst.data }}`

**Comment by jaymedina** [comment]
@hover2pi thanks! Just pushed a final commit-- once the Travis build finishes, this should be good to go.

**Pull request merged**